libraries/animation/src/AnimInverseKinematics.cpp

```diff
@@ -158,6 +159,16 @@ void AnimInverseKinematics::setTargetVars(const QString& jointName, const QStrin
     }
 }
 
+bool debounceJointWarnings() {
+    static QTime last_call;
```

Comment:
Would probably be better to declare this QTime near the start of the file, e.g., directly after `JOINT_WARNING_DEBOUNCE_TIME`.
Also, name the object, say, `debounceJointWarningsClock` - variable name with "_" is non-standard.